doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -196,8 +228,8 @@ namespace Azure.AI.MetricsAdvisor
         // Suggest appending values to a plural method name
         public virtual AsyncPageable<MetricFeedback> GetMetricFeedbacksValuesAsync(Guid feedbackId, CancellationToken cancellationToken = default)
         {
-            // Call protocol method
-            AsyncPageable<BinaryData> pageableBinaryData = GetMetricFeedbacksAsync(feedbackId, new RequestContext() { CancellationToken = cancellationToken });
+            // Call internal paging implementation by passing the new scope name, the inner scope will be suppressed
```

Comment:
Lets also add here that we need to pass this because the convenience method has a different name from the protocol method.